### PR TITLE
WIP: cleanup

### PR DESCRIPTION
**Pull request by rhaschke**
Several improvements to the ROS2 code migration. Additional changes are required:
- [use of `rclcpp::Clock` for `ros::Time`](https://answers.ros.org/question/287946/ros-2-time-handling)
- ...

### CODE REVIEW

File: moveit_core/background_processing/src/background_processing.cpp [hunk 1]
```diff
@@ -35,14 +35,14 @@
...
 #include <moveit/background_processing/background_processing.h>
-#include "rclcpp/rclcpp.hpp"
+#include <rclcpp/logging.hpp>
 
 namespace moveit
 {
 namespace tools
 {
 // Logger
-static const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit_background_processing.background_processing");
```
Comment (henningkayser) [inline moveit_core/background_processing/src/background_processing.cpp]: We should use a systematic naming convention for this (see https://github.com/ros-planning/moveit2/blob/master/doc/MIGRATION_GUIDE.md). Right now we decided to use the library name only (here `moveit_background_processing`) without a separator if the file name is identical. I agree that having a separator would make sense for scoping logger levels. Also, I would argue for including the package name, i.e. `moveit.core.background_processing` following the pattern `moveit.<package>.<library>.<file>` where we drop `moveit` from `<package>.<library>.<file>` and remove redundancies in the namespace if there are any. What do you think of this approach?

File: moveit_core/background_processing/src/background_processing.cpp [hunk 2]
```diff
@@ -35,14 +35,14 @@
 /* Author: Ioan Sucan */
 
 #include <moveit/background_processing/background_processing.h>
-#include "rclcpp/rclcpp.hpp"
```
Comment (henningkayser) [inline moveit_core/background_processing/src/background_processing.cpp]: Yes, this was also on my list ;)

File: moveit_core/background_processing/src/background_processing.cpp (hunk 1 above)
Comment (rhaschke) [reply moveit_core/background_processing/src/background_processing.cpp]: Adding the package name is a good idea. My main aim was to introduce the top-level `moveit` scope.

**Comment by mlautman** [comment]
correction, removing that line doesn't fix it. You may need to do something else

**Comment by RoboticsYY** [comment]
Hi @rhaschke, I noticed this PR is still titled as WIP. Are you planning to continue this PR? Otherwise, I would like to continue this effort.

**Comment by rhaschke** [comment]
@RoboticsYY, please feel free to continue. I don't have the time to work on ROS2.

**Comment by rhaschke** [comment]
Closing in favor of #170.

**Pull request closed without merging**